index.html

```diff
@@ -27,7 +27,7 @@
         <meta charset="UTF-8">
 
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.6.1/dat.gui.min.js"></script>
+        <script src="node_modules/dat.gui/build/dat.gui.min.js"></script>
```

Comment:
If you expect people using `index.html` to install `devDependencies` then this must be documented in `README.md` or `BUILDING.md`.